runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -1914,6 +1914,25 @@ TR_J9ServerVM::getObjectSizeClass(uintptr_t objectSize)
    return 0;
    }
 
+bool
+TR_J9ServerVM::acquireClassTableMutex()
+   {
+   // Acquire per-client class table lock
+   TR::Monitor *classTableMonitor = _compInfoPT->getClientData()->getClassMapMonitor();
```

Comment:
We need the acquire the new monitor that was created for the CHTable per client

Reply:
Yeah, this is an unexpected error. I first had the CH table monitor in client session but then I realized that it's better to have it inside server-side CH-table and moved it there. But I also changed the name from `_classTableMonitor` to `_classMapMonitor` without realizing that the monitor with such name already exists.